### DISCUSSION TITLE
Set default fold level

### DISCUSSION THREAD

**Discussion by DonnieWest** [NeogitOrg/neogit#1635] [Ideas]
I'd appreciate being able to set a default fold level. A ton of my flow end up expanding all of the folds by default. It might also be possible to read VIM's native foldlevel value initially

This is possibly related to #10

**Comment by nagy135** [comment]
not sure if this is related, but every time i open `NeogitCommitView` everything is expanded. I think that isnt good default (or at least should be configurable). When I open commit, first of all I wanna see files that are affected. Then hunks and their contents. Probably bind that folds everything would be sufficient. If I remember correctly, emacs have this "global folding feature" where tab/shift+tab folds everything, something like that would be amazing.

**Comment by kunzaatko** [comment]
Hi @DonnieWest!
You can fix this by setting the native `foldlevel` option, as you suggested, through an autocommand. This is the related part of my config, which I run when loading the plugin:

```lua
vim.cmd [[
    autocmd FileType NeogitStatus,NeogitPopup lua vim.opt_local.spell = false; vim.opt_local.list = false; vim.opt_local.foldlevel = 1000
]]
```
